README.md

```diff
@@ -130,6 +130,9 @@ The following configuration options are available:
 - `--labeled-output-topics`: Additional labeled output topics if you need to specify multiple topics with different
   message types (`String=String>[,<String=String>...]`)
 
+- `--group-id`: Unique application ID to use for Kafka Streams. Can also be provided by
+  implementing `StreamsApp#getUniqueAppId()`
```

Comment:
Hm, I would say yes because the Streams config is called application.id. Maybe it does make more sense to use --application-id after all? The idea with --group-id was just to use the same parameter as one would for a plain consumer app

Reply:
Hm, maybe sticking with appld is better

Reply:
Ok, I decided to use application id. It makes the most sense in context of Kafka Streams and we don't have any pure consumer apps right now